README.md

```diff
@@ -0,0 +1,58 @@
+4. Download the photo naming script available at [`resources/tools/namePictures.py`](https://github.com/apc-gmhs/seniormap/blob/gh-pages/resources/tools/namePictures.py).
+5. Convert the spreadsheet of names into a 2D array to replace `[['firstname', 'lastname']]`.
+   - For example `[['Bill', 'Axel'], ['Sally', 'Breeze'], ['Manuel','Collins']]`
+   - There's a number of online tools that will do this for you, such as [this site](https://www.seabreezecomputers.com/excel2array).
+6. Run the Python script, renaming all of the unnamed photos to their corresponding student names. 
+   - It's important to note that both the array of names and the student photos must be aligned and set up correctly to function properly. For example if a student is present in the array, but doesn't have a photo, all of the photo's names will be off by one. Setting up the script correctly is a little bit time consuming, but far faster than manually renaming each photo.
+7. Create a new folder inside of `portraits` named with the corresponding year.
+8. Upload the pictures to that folder.
+
+### Adding a new student to the map after form submission 
+
+1. After a student submits the form, a new entry will appear in the the corresponding spreadsheet.
+2. If the student appears on the map, highlight the entry in green to indicate they're visible.
+3. If the student does not appear on the map, the college they are attending must be added .
+   - To do this, create a new entry in 'Shared Data' (Senior Map/Shared Data/Location Information) for both the coordinates and logos subsheets.
+   - Mimicking existing cells, add the coordinates, and the logo to the entries.
+   ![Screenshot of logo cells](resources/readme/tut3.png)
+   ![Screenshot of logo cells](resources/readme/tut4.png)
+   - A good way to see if certain schools don't have logos is by using the developer console. The names of institutions missing logos will appear here.
+   ![Screenshot of logo cells](resources/readme/tut2.png)
+4. After ensuring the student appears, mark them visible!
+
+### Passing the map onto future classes 
+1. Find an experienced leadership member of the APC who is a student in the next graduating class who is able and willing to take over the map.
+2. Request their personal email address and transfer ownership of the Senior Map Google Cloud Project to them.
+3. Give their student email account ownership of all Senior Map Google Drive folders and files.
+4. Make sure they can access the Senior Map GitHub repository as editor.
+
+---
+
+If you have any questions about the codebase/maintaining it, or adding a new class to the page, don't hesistate to reach out to past maintainers. Also make sure you publicize the map to your class to carry on the tradition! Talk to your class officers about posting the map on Schoology, reach out to your college counselor about sending the map out and consider sharing the map on social media.
```

Comment:
Might be worthwhile to add a list of previous maintainers and their emails/GH accounts

Reply:
I'll link the Github account - if future maintainers want to have an email available it can always be listed on their GitHub account profile page.

Reply:
I added you too which I hope was okay -- lmk if you want to be removed